README.md

```diff
@@ -322,6 +322,7 @@ Adjust Less styling by over-riding Less Variables in the parent application.
 // Widget selector
 @impac-dashboard-widget-selector-bg:            #233845;
 @impac-dashboard-widget-selector-text-color:    white;
+@impac-dashboard-widget-selector-widget-item-min-height: 40px;
```

Comment:
Re my comment: I think the min-height could be adjusted to be high enough for 2 lines (maybe 60px?...)
Otherwise, good idea to put that in a variable.